omod/src/main/webapp/pages/summarydashboard/programEnrollments.gsp

```diff
@@ -0,0 +1,91 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage")
+    ui.includeCss("coreapps", "clinicianfacing/patient.css")
+%>
+
+<script type="text/javascript">
+	var breadcrumbs = [
+		{ icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+		{ label: "${ ui.message(app.label)}"}
+	];
+
+	jq(function(){
+		jq(".tabs").tabs();
+		// make sure we reload the page if the location is changes; this custom event is emitted by the location selector in the header
+		jq(document).on('sessionLocationChanged', function() {
+			window.location.reload();
+		});
+	});
+</script>
+
+<div class="clear"></div>
+<div class="container">
+	<div class="dashboard clear">
+		<!-- only show the title div if a title has been defined in the messages.properties -->
+		<% if (ui.message(app.id + ".custom.title") != app.id + ".custom.title") { %>
+		<div class="title">
+			<h3>${ ui.message(app.id + ".custom.title") }</h3>
+		</div>
+		<% } %>
```

Comment:
Is  what @mseaton  said true?  I agree we wouldn't want to take first column fragments designed for another page/app and display them here, but it looks like the columns are defined by the name of this particular app, though I could be forgetting how it all works or missing something obvious.  fyi @cioan

Reply:
Thanks @mogoodrich ! I am not sure how all this works either. I thought I could just use the current CustomAppLoaderFactory configuration to display the already configured program statistics widget here. But since this is a new page, I could just remove this type of on the fly configuration, and just hard code the program statistics widget at the top of the page and the table of enrollments underneath. Would that be better? That would mean removing all the existing program summary configuration from the CustomAppLoaderFactory because it would be pointless if we hard coded in the new page?

Reply:
@cioan  do you have a branch with the CALF with the changes you made?  It will likely be easier for me to review/understand from that.

Reply:
@mogoodrich : https://github.com/PIH/openmrs-module-pihcore/pull/524